pkg/gardenlet/controller/federatedseed/networkpolicy/namespace_control.go

```diff
@@ -107,15 +106,13 @@ func (r *namespaceReconciler) Reconcile(request reconcile.Request) (reconcile.Re
 	}
```

Comment:
Can you check that if statement here?
I think its superfluous, as `err` is always `nil` here.